src/content/docs/principles.md

```diff
@@ -9,11 +9,11 @@ These principles are not cast in stone, but are strong guidelines for developers
 
 Above all, ICU4X must provide modern, standards-compliant APIs that encourage best internationalization practices and produce correct results for all languages and locales. No language or locale should be at a structural disadvantage.
 
-## Code Style
+## Code practices
```

Comment:
Are the changes here being made manually or are they the result of rerunning the MD conversion script?

Reply:
reverted